openshift/e2e-tests-openshift.sh

```diff
@@ -324,11 +366,15 @@ function delete_istio_openshift(){
   oc delete ControlPlane/basic-install -n istio-system
 }
 
-function delete_serving_openshift() {
-  echo ">> Bringing down Serving"
-  oc delete --ignore-not-found=true -f serving-resolved.yaml
+function delete_knative_openshift() {
+  echo ">> Bringing down Knative Serving, Build and Pipeline"
+  oc delete --ignore-not-found=true -n $OLM_NAMESPACE -f knative-serving.catalogsource-ci.yaml
```

Comment:
I don't think this will actually delete the subscription or its associated operator, but you may not care.

Reply:
Nope, don't think we care a lot here. Not even sure why we try to be so eloquent in teardown.